### PR TITLE
Build application without ops center

### PR DESCRIPTION
**Pull request by UnderGreen**

### CODE REVIEW

**Review by wadells — COMMENTED** [review]
The `build-app` target requires both `download-binaries` and `install-dependent-packages` to function correctly (and these need to be re-run every time `GRAVITY_VERSION` is touched) -- but it doesn't depend on either of them.   While this works fine for CI, since it always has a clean start, it is likely to cause trouble for maintainers who switch between different branches in one repo locally.

This is annoying becasue the error message is a cryptic:

```
/home/walt/git/stolon-app/bin/tele build -f -o build/installer.tar --name=stolon-app --version=1.13.9-dev.24 --glob=**/*.yaml  --parallel=1 --set-image=stolon-bootstrap:1.13.9-dev.24 --set-i
mage=stolon-uninstall:1.13.9-dev.24 --set-image=stolon-hook:1.13.9-dev.24 --set-image=stolon:1.13.9-dev.24 --set-image=stolon-telegraf:1.13.9-dev.24 --set-image=stolonctl:1.13.9-dev.24 --set
-image=stolon-pgbouncer:1.13.9-dev.24 --set-image=stolon-etcd:1.13.9-dev.24 --set-image=stolon-common:1.13.9-dev.24  --state-dir=state build/resources/app.yaml                               
Thu Feb 11 23:48:07 UTC Building cluster image stolon-app 1.13.9-dev.24                                                                                                                       
Thu Feb 11 23:48:07 UTC Selecting base image version                                           
        Will use base image version 7.0.16                                                     
Thu Feb 11 23:48:07 UTC Syncing packages for 7.0.16                                                                                                                                           
Thu Feb 11 23:48:07 UTC Downloading dependencies from https://get.gravitational.io             
Thu Feb 11 23:48:09 UTC Build finished in 2 seconds                                            
[ERROR]: failed to sync packages for runtime version 7.0.16                                    
        package(gravitational.io/cluster-ssl-app:0.8.4) not found 
```

When what it really means is "you're using old binaries, run `download-binaries` again and check your path".  I'd strongly recommend fixing the dependency relationship, or at least adding an note to the README that the targets must be run in order.

File: Makefile
Comment (wadells) [inline Makefile:9]: We can probably cut the path munging in the .drone.yml/Jenkinsfile if we update these to reference `./bin/...`

Comment (UnderGreen) [reply Makefile:9]: I can't do that. For example, I am using Nix and I created shell.nix to download binaries and put them in nix store from which I can point them later if needed.

File: .drone.yml
Comment (wadells) [inline .drone.yml:31]: This requires re-signing -- otherwise the tests will need manual approval.

**Review by wadells — APPROVED** [review]

**Pull request merged**
